Content.Shared/Construction/ConstructionGraphNode.cs

```diff
@@ -31,6 +31,17 @@ public sealed partial class ConstructionGraphNode
         [DataField("entity", customTypeSerializer: typeof(GraphNodeEntitySerializer), serverOnly:true)]
         public IGraphNodeEntity Entity { get; private set; } = new NullNodeEntity();
 
+        /// <summary>
+        ///     Ignore requests to change the entity if the entity's current prototype inherits from specified replacement 
+        /// </summary>
+        /// <remarks>
+        ///     When this bool is true and a construction node specifies that the current entity should be replaced with a new entity,
+        ///     if the current entity has an entity prototype which inherits from the replacement entity prototype,
+        ///     entity replacement will not occur 
```

Comment:
A comment documenting when this could be useful (e.g. airlock hacking protections) would be helpful (and specifically explaining why it's useful).

Reply:
Done